openmc/material.py

```diff
@@ -56,6 +56,9 @@ class Material(object):
         Units used for `density`. Can be one of 'g/cm3', 'g/cc', 'kg/cm3',
         'atom/b-cm', 'atom/cm3', 'sum', or 'macro'.  The 'macro' unit only
         applies in the case of a multi-group calculation.
+    depletable : bool
```

Comment:
Good point -- I'll add it to XML/summary. It would be possible to use `openmc.data` to automatically identify fissionable materials but that might mean opening up a lot of HDF5 files.